rpm/herokuish.postinst

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set +o posix # rpm uses /bin/sh that forces bash in posix mode
+
+echo 'Starting docker'
+systemctl start docker
+
+sleep 5
+
+count=`sudo docker images | grep gliderlabs/herokuish | wc -l`
```

Comment:
This was taken from https://github.com/dokku/dokku/blob/master/deb.mk#L87-L95. Do we need to change that also in deb packages?